<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPublishStreamClass</class>
 <widget class="QMainWindow" name="QPublishStreamClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>696</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>873</width>
    <height>589</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QPublishStream</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="m_widget_video" native="true"/>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="m_lab_volume_2">
               <property name="text">
                <string>address:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="m_edit_address"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="m_btn_start_publishstream">
                 <property name="text">
                  <string>play</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="m_btn_pause">
                 <property name="text">
                  <string>Pause</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="m_btn_stop_publishstream">
                 <property name="text">
                  <string>stop</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="m_slider_seek">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="m_lab_volume">
                 <property name="text">
                  <string>volume</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="m_slider_volume">
                 <property name="value">
                  <number>99</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>rate</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="m_slider_playrate">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>12</number>
                 </property>
                 <property name="value">
                  <number>6</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
